releases/nightly-build/compose-files/docker-compose-nexus-redis-no-secty.yml

```diff
@@ -231,14 +253,16 @@ services:
     depends_on:
       - export-client
     environment:
-      - EXPORT_DISTRO_CLIENT_HOST=export-client
-      - EXPORT_DISTRO_DATA_HOST=edgex-core-data
-      - EXPORT_DISTRO_CONSUL_HOST=edgex-config-seed
-      - EXPORT_DISTRO_MQTTS_CERT_FILE=none
-      - EXPORT_DISTRO_MQTTS_KEY_FILE=none
+      <<: *common-variables
+      EXPORT_DISTRO_CLIENT_HOST: export-client
+      EXPORT_DISTRO_DATA_HOST: edgex-core-data
+      EXPORT_DISTRO_CONSUL_HOST: edgex-config-seed
+      EXPORT_DISTRO_MQTTS_CERT_FILE: none
+      EXPORT_DISTRO_MQTTS_KEY_FILE: none
 
   rulesengine:
-    image: nexus3.edgexfoundry.org:10004/docker-support-rulesengine:1.1.0
+    # for now- use version 1.0.0 until 1.1.0 available
+    image: nexus3.edgexfoundry.org:10002/docker-support-rulesengine:1.0.0
```

Comment:
nexus3.edgexfoundry.org:10004/docker-support-rulesengine:1.1.0-dev.1 is now available.